packages/client/components/NewMeeting.tsx

```diff
@@ -38,37 +42,32 @@ const TeamAndSettings = styled('div')<{isDesktop}>(({isDesktop}) => ({
   flexDirection: 'column',
   gridArea: 'settings',
   marginTop: isDesktop ? 32 : undefined,
-  [MEDIA_QUERY_VERTICAL_CENTERING]: {
-    minHeight: isDesktop ? undefined : 166
-  }
+  minHeight: 166
 }))
 
 const TeamAndSettingsInner = styled('div')({
   borderRadius: '4px',
   boxShadow: Elevation.Z1
 })
 
-const NewMeetingBlock = styled('div')<{innerWidth: number; isDesktop: boolean}>(
-  {
-    alignItems: 'flex-start',
-    backgroundImage: 'linear-gradient(0deg, #F1F0FA 25%, #FFFFFF 50%)',
-    backgroundRepeat: 'no-repeat',
-    display: 'flex',
-    flexDirection: 'column',
-    justifyItems: 'center',
-    minHeight: '100%'
-  },
-  ({innerWidth, isDesktop}) =>
-    isDesktop && {
-      backgroundImage: `url('${WaveSVG}'), linear-gradient(0deg, #F1F0FA 50%, #FFFFFF 50%)`,
-      backgroundSize: '100%',
-      // the wave is 2560x231, so to figure out the offset from the center, we need to find how much scaling there was
-      backgroundPositionY: `calc(50% - ${Math.floor(((innerWidth / 2560) * 231) / 2 - 1)}px), 0`,
-      height: '100%',
-      minHeight: 0,
-      overflow: 'auto'
-    }
-)
+const NewMeetingDialog = styled(DialogContainer)({
+  width: '800px',
+  maxHeight: 'unset'
```

Comment:
-1 we never want a modal to be taller than the viewport. in the pic below, the CTA is below the fold, which is no good. instead.

options:
- shrink/hide the image when the instructions are expanded
- make the instructions overflow: auto
- cut some margin between the options & "1 Active Meeting"
- ???

![Screenshot from 2022-07-15 16-20-13](https://user-images.githubusercontent.com/5514175/179323692-284083e9-6391-4768-b818-14b4a908a649.png)

Reply:
I agree the CTA shouldn't be below the fold, but I think in this case it would be acceptable since:
1) it only moves out of screen once the user interacts with the "How to Run" section
2) it simplifies scrolling for the user since the whole screen is the scrolling area
3) the whole content will be changed soon anyways

Reply:
@mattkrick what do you think about my comment above?

Reply:
From a UX perspective, we just can't hide a primary CTA. The primary purpose of this modal is to get the user to click that button, so if we hide it, we're defeating that purpose. In this screen, you can see the CTA peek out a little bit, but on a slightly smaller screen, they wouldn't even see the button & there's no hint that they're supposed to scroll.

We could remove the illustration when they click "how to run", remove "how to run" once it's been clicked, condense the margin of the 4 steps, "learn more" and "1 active meeting". all of those would be preferred over hiding the CTA

Reply:
or we could just remove "how to run" on mobile altogether!

Reply:
Let's remove the "How to Run" if there is not enough space. If we run into this situation with the next iteration of new meeting dialog, we should place some arrow on the bottom or similar to indicate there is scrolling needed (actually I see a scroll bar on your screenshot, but that might be missed).

I think it might be better to make the dialog fullscreen on mobile
* we have less space wasted as the background is not of any value anyways
* the scrollbar would be visible on top of the dialog